read-fonts/src/tables/fvar.rs

```diff
@@ -92,4 +96,23 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn normalize_overflow() {
```

Comment:
We’ve been adding a comment with links to the oss-fuzz and chromium bugs to tests designed to capture them.

Reply:
Link added.